ports.php

```diff
@@ -178,8 +178,7 @@
                  $row = pg_fetch_assoc($result), $i++) {
 
                 /* Port name and Portfile URL */
-                print "<dt><b><a href=\"${trac_url}browser/trunk/dports/" . $row['path'] . "/Portfile\">" . htmlspecialchars($row['name'])
-                . '</a></b> ' . htmlspecialchars($row['version']) . '</dt>';
+                print '<dt><b>' . htmlspecialchars($row['name']) . '</b> ' . htmlspecialchars($row['version']) . "<i>(<a href=\"${github_url_ports}blob/master/" . $row['path'] . "/Portfile\">source</a>)</i></dt>";
```

Comment:
I can't remember right now whether PHP supports the ${} variable syntax, but I do remember that values of href attributes should be encoded with htmlspecialchars, so please make this

```php
... href=\"" . htmlspecialchars($github_url_ports . 'blob/master' . $row['path'] . '/Portfile') . "\" ...
```